core/templates/dev/head/pages/topic-viewer-page/topic-viewer-page.controller.ts

```diff
@@ -65,14 +65,14 @@ angular.module('oppia').directive('topicViewerPage', [
 
             $rootScope.loadingMessage = 'Loading';
             TopicViewerBackendApiService.fetchTopicData(ctrl.topicName).then(
-              function(topicDataDict) {
-                ctrl.topicId = topicDataDict.topic_id;
-                ctrl.canonicalStoriesList = topicDataDict.canonical_story_dicts;
-                ctrl.degreesOfMastery = topicDataDict.degrees_of_mastery;
-                ctrl.skillDescriptions = topicDataDict.skill_descriptions;
-                ctrl.subtopics = topicDataDict.subtopics;
+              function(topicDataObject) {
```

Comment:
Don't end the variable name for an object with `Object`, we only add Dicts at the end to explain the variable contains dict object.

Reply:
I think only `topicData` sounds good